modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -158,7 +162,7 @@ const TooltipProvider = () => {
 				document.body,
 				eventName,
 				SELECTOR_TRIGGER,
-				(event) => {
+				(event: MouseEvent & {delegateTarget: HTMLElement}) => {
```

Comment:
This is the only sadness in the PR for me, because `delegateTarget` is a non-standard property. But given that we are using `delegate` from `frontend-js-web`, and I doubt that is going to change any time soon, I guess we don't have a choice to continue working like this.